src/explore-education-statistics-common/src/modules/release/components/__tests__/ReleaseDataAndFiles.test.tsx

```diff
@@ -0,0 +1,85 @@
+import ReleaseDataAndFiles from '@common/modules/release/components/ReleaseDataAndFiles';
+import { Release } from '@common/services/publicationService';
+import { render, screen } from '@testing-library/react';
+import React from 'react';
+import { FileInfo } from '@common/services/types/file';
+
+describe('ReleaseDataAndFiles', () => {
+  const testRelease = {
+    id: 'r1',
+    downloadFiles: [
+      {
+        id: 'f1',
+        name: 'All files',
+        type: 'Ancillary',
+      },
+      {
+        id: 'f2',
+        name: 'data file',
+        type: 'Data',
+      },
+      {
+        id: 'f3',
+        name: 'ancillary file',
+        type: 'Ancillary',
```

Comment:
Could use more unique names i.e. 'Test data file' and 'Test ancillary file'. The risk of using less unique names is that text assertions could pick these pieces of text by accident if, for example, the guide text is changed in the future.